cnf-tests/testsuites/e2esuite/bond/bond.go

```diff
@@ -20,15 +20,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = Describe("[bondcni]", func() {
+var _ = Describe("[bondcni]", Ordered, func() {
 	apiclient := client.New("")
 
 	execute.BeforeAll(func() {
 		err := namespaces.Create(namespaces.BondTestNamespace, apiclient)
 		Expect(err).ToNot(HaveOccurred())
 	})
 
-	AfterEach(func() {
+	AfterAll(func() {
```

Comment:
Does it work? In this suite, we have only one test case, so AfterAll will be called at a very similar timing to AfterEach, doesn't it?

Reply:
you're right. I didn't test it and was sure the ReportAfterEach will come before the AfterAll, but in reality, it's not.
So I'll fix it by adding the last cleanup in the last test as we had before.